packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -162,7 +173,7 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
       streamId
     )
 
-    if (payload.header) {
+    if (payload.header) {ß
```

Comment:
there's an extra character here
```suggestion
    if (payload.header) {
```

Reply:
Thanks for catching this